hoomd/md/EvaluatorPairExpandedGaussian.h

```diff
@@ -0,0 +1,197 @@
+            v["epsilon"] = epsilon;
+            v["delta"] = delta;
+            return v;
+            }
+#endif
+        }
+
+        __attribute__((aligned(16)));
+
+    //! Constructs the pair potential evaluator
+    /*! \param _rsq Squared distance between the particles
+        \param _rcutsq Squared distance at which the potential goes to 0
+        \param _params Per type pair parameters of this potential
+    */
+    DEVICE EvaluatorPairExpandedGaussian(Scalar _rsq, Scalar _rcutsq, const param_type& _params)
+        : rsq(_rsq), rcutsq(_rcutsq), epsilon(_params.epsilon), sigma(_params.sigma),
+          delta(_params.delta)
+        {
+        }
+
+    //! Gauss doesn't use diameter
+    DEVICE static bool needsDiameter()
+        {
+        return false;
+        }
+    //! Accept the optional diameter values
+    /*! \param di Diameter of particle i
+        \param dj Diameter of particle j
+    */
+    DEVICE void setDiameter(Scalar di, Scalar dj) { }
+
+    //! Gauss doesn't use charge
+    DEVICE static bool needsCharge()
+        {
+        return false;
+        }
+    //! Accept the optional diameter values
+    /*! \param qi Charge of particle i
+        \param qj Charge of particle j
+    */
+    DEVICE void setCharge(Scalar qi, Scalar qj) { }
+
+    //! Evaluate the force and energy
+    /*! \param force_divr Output parameter to write the computed force divided by r.
+        \param pair_eng Output parameter to write the computed pair energy
+        \param energy_shift If true, the potential must be shifted so that V(r) is continuous at the
+       cutoff \note There is no need to check if rsq < rcutsq in this method. Cutoff tests are
+       performed in PotentialPair.
+
+        \return True if they are evaluated or false if they are not because we are beyond the cutoff
+    */
+    DEVICE bool evalForceAndEnergy(Scalar& force_divr, Scalar& pair_eng, bool energy_shift)
+        {
+        // precompute some quantities
+        Scalar r = fast::sqrt(rsq);
+        Scalar rcut = fast::sqrt(rcutsq);
+        Scalar rinv = Scalar(1.0) / r;
+
+        // compute the force divided by r in force_divr
+        if (rsq < rcutsq)
+            {
+            Scalar rmd = r - delta;
+            Scalar rmd_sq = (r - delta) * (r - delta);
+            Scalar sigma_sq = sigma * sigma;
+            Scalar rmd_over_sigma_sq = rmd_sq / sigma_sq;
+            Scalar exp_val = fast::exp(-Scalar(1.0) / Scalar(2.0) * rmd_over_sigma_sq);
+
+            force_divr = epsilon / sigma_sq * exp_val * rmd * rinv;
+            pair_eng = epsilon * exp_val;
+
+            Scalar rcutmd_sq = (rcut - delta) * (rcut - delta);
+            Scalar rcutmd_over_sigma_sq = rcutmd_sq / sigma_sq;
+
+            if (energy_shift)
+                {
+                pair_eng -= epsilon * fast::exp(-Scalar(1.0) / Scalar(2.0) * rcutmd_over_sigma_sq);
+                }
+            return true;
+            }
+        else
+            return false;
```

Comment:
```suggestion
        else
            {
            return false;
            }
```
I think the hoomd syntax policy is to include open/closed brackets even if the if/else statement is only one line, but I am unsure about this.

Reply:
Yes, this is good practice, though not enforced. We could check and see if `clang-format` has an option to enforce this.